app/app.go

```diff
@@ -482,9 +487,26 @@ func NewEvmos(
 		app.IBCKeeper.ChannelKeeper, &app.IBCKeeper.PortKeeper,
 		app.AccountKeeper, app.BankKeeper, scopedTransferKeeper,
 	)
+
+	app.WithdrawKeeper = *withdrawkeeper.NewKeeper(
+		app.GetSubspace(withdrawtypes.ModuleName),
+		app.AccountKeeper,
+		app.BankKeeper,
+		app.ClaimsKeeper.Hooks(), // TODO: verify ics4 wrapper
```

Comment:
this should be the `ChannelKeeper`? This is the top level of the stack correct? The withdraw keeper should be the ICS4Wrapper for the ClaimsKeeper

Reply:
still not sure how the ICS04 is set up